docs/docs/introduction.md

```diff
@@ -27,15 +27,15 @@ LSMD runs under a popular message brokering protocol called MQTT. We contribute
 3. To make the broker aware of the computational latency of processing units, we implemented the watermark-based back-pressure[], a mechanism which indicates whether a processing unit is overloaded. With these extensions of MQTT, the latency-sensitive broker can detect the latency conditions of the processing units. The LSMD is able to dispatch workload to each consumer group based on the latency-related metrics.
 
 ### Consumer Group
-<img src="https://dsngroup.github.io/broke/docs/site/build/broke/img/consumergroup.png" alt="Drawing" style="width: 350px;"/>
+<img src="https://dsngroup.github.io/broke/img/consumergroup.png" alt="Drawing" style="width: 350px;"/>
```

Comment:
A small nit, can this resolve with relative path?

Reply:
Because we use auto-deployment to ```gh-pages``` now, we didn't need to build the website manually. Then the files in ```/build``` is not needed anymore. You can take a look at ```gh-pages``` branch.

Reply:
I misunderstood your point lol. I'll give it a try